### PR TITLE
add adtk

### PR DESCRIPTION
**Pull request by rpanai**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by rpanai** [comment]
What could I do to fix this recipe? If I don't specify `python >=3.5` it fails cos of python 2.7 but if I do set the python version I got a linter error. @bletham Do you mind to offer some lead?
I'm eventually planning to PR to the original library [here](https://github.com/arundo/adtk) as skeleton doesn't  seem to work with requirements in `setup.cfg`.

File: recipes/adtk/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 8424fff325f9e0fc60db15d108eeef9d8e32ca2ea2d7d8c398ccb712cde1c905
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win]
```
Comment (scopatz) [inline recipes/adtk/meta.yaml]: Can this be `noarch: python` instead of `skip: true  # [win]`

Comment (rpanai) [reply recipes/adtk/meta.yaml]: Thanks. I still have this `SSL` error

```
Traceback (most recent call last):
  File "C:\projects\staged-recipes\ff_ci_pr_build.py", line 201, in <module>
    sys.exit(main())
  File "C:\projects\staged-recipes\ff_ci_pr_build.py", line 192, in main
    exit_code = int(appveyor_check_latest_pr_build(repo, pr, bld) is False)
  File "C:\projects\staged-recipes\ff_ci_pr_build.py", line 117, in appveyor_check_latest_pr_build
    data = request_json(url.format(repo=repo, total_builds=total_builds), headers=headers)
  File "C:\projects\staged-recipes\ff_ci_pr_build.py", line 46, in request_json
    with contextlib.closing(urlopen(request)) as response:
  File "C:\Python27\lib\urllib2.py", line 154, in urlopen
    return opener.open(url, data, timeout)
  File "C:\Python27\lib\urllib2.py", line 429, in open
    response = self._open(req, data)
  File "C:\Python27\lib\urllib2.py", line 447, in _open
    '_open', req)
  File "C:\Python27\lib\urllib2.py", line 407, in _call_chain
    result = func(*args)
  File "C:\Python27\lib\urllib2.py", line 1241, in https_open
    context=self._context)
  File "C:\Python27\lib\urllib2.py", line 1198, in do_open
    raise URLError(err)
urllib2.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:727)>
Command exited with code 1
```

Comment (scopatz) [reply recipes/adtk/meta.yaml]: you don't need to worry about appveyor if azure on windows works

File: recipes/adtk/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  host:
+    - pip
+    - python >=3.5
+    - pandas >=0.23
+    - numpy >=1.15
+    - matplotlib >=3.0
+    - scikit-learn >=0.20
+    - statsmodels >=0.9
+    - packaging >=17.0
+    - tabulate >=0.8
```
Comment (scopatz) [inline recipes/adtk/meta.yaml]: Do you actually need most of these to install the package? Or do you just need Python and pip?

Comment (rpanai) [reply recipes/adtk/meta.yaml]: Thes packages are listed [here](https://github.com/arundo/adtk/blob/develop/setup.cfg)

Comment (scopatz) [reply recipes/adtk/meta.yaml]: right, but that is runtime deps, not build time. I don't think you need these

Comment (rpanai) [reply recipes/adtk/meta.yaml]: Done!

**Pull request closed without merging**